packages/utils/internal-utils/src/variants/index.ts

```diff
@@ -1,12 +1,8 @@
-import { Picks, VariantLookup } from './types'
+import { VariantLookup } from './types'
 
-/* eslint-disable space-before-function-paren */
-function makeVariants<
-  Variant extends 'design' | 'intent' | 'shape' | 'size',
-  P extends (keyof VariantLookup[Variant])[] = []
->(
-  variants: P extends [] ? VariantLookup[Variant] : Picks<VariantLookup[Variant], P>
-): P extends [] ? VariantLookup[Variant] : Picks<VariantLookup[Variant], P> {
+function makeVariants<Variant extends 'design' | 'intent' | 'shape' | 'size'>(
+  variants: Partial<VariantLookup[Variant]>
```

Comment:
Let me know what you think @acd02 🙏🏻

Reply:
LGTM, it's less cumbersome this way 👍 

But be sure to update every occurrence of that function call 😅